### PR TITLE
Dev merge create workouts

### PR DESCRIPTION
**Pull request by ctperry0301**
With this being implemented, we can delete DEV-7 (Rohiths) and DEV-8 (Coles) parts

### CODE REVIEW

**Comment by rohithdara** [comment]
Looks fire, but I'm biased so I don't think I should be reviewing this lol

File: components/setWorkoutDetails/SetWorkoutDetailsCard.jsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  box-shadow: 3px 5px 2px #00000050;
+`;
+
+const SetWorkoutDetailsCard = (props) => {
+  const {
+    color, name, displayEllipses, sets, setSets, seconds, setSeconds, reps, setReps, removeExercise
+  } = props;
+  //const [sets, setSets] = React.useState('');
+  //const [reps, setReps] = React.useState('');
+  //const [seconds, setSeconds] = React.useState('');
```
Comment (jpoist97) [inline components/setWorkoutDetails/SetWorkoutDetailsCard.jsx]: nit: delete these

File: components/setWorkoutDetails/SetWorkoutDetailsCard.jsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          options={[
+            {
+              icon: 'SWAP', text: 'Switch Reps/Time', onPress: () => (isReps ? setIsReps(false) : setIsReps(true)),
+            },
+            {
+              icon: 'DELETE', text: 'Remove Exercise', onPress: () => {removeExercise()},
+            }]}
+        />
+      ) : <View />}
+      {/* TODO: Delete should remove exercise from workout */}
```
Comment (jpoist97) [inline components/setWorkoutDetails/SetWorkoutDetailsCard.jsx:79]: can delete this too

File: components/workouts/CreateWorkout.jsx [hunk 3]
```diff
@@ -34,6 +35,47 @@ const AddCycleButton = styled(PlusButton)`
...
+        setItemState(newItemState);
+    }
+    const colors = ['#CAB0FF', '#9D8DFF', '#6D8DFF'];
+    const onExercisesAdd = (selectedExercises) => {
+      const newItems = [...itemState];
+      newItems.push(...selectedExercises)
+      const newExercise = newItems.map((item) => {
+          item.color = colors[(newItems.indexOf(item) + 3) % 3];
+          return item;
+      })
```
Comment (jpoist97) [inline components/workouts/CreateWorkout.jsx]:
```suggestion
      const newExercise = newItems.map((item, index) => {
          item.color = colors[index % 3];
          return item;
      })
```

File: package.json [hunk 4]
```diff
@@ -28,6 +28,7 @@
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz",
     "react-native-alphabet-sectionlist": "0.0.4",
+    "react-native-elements": "^3.0.0-alpha.1",
```
Comment (jpoist97) [inline package.json:31]: are we meant to have the alpha version of this package? @rohithdara

**Review by jpoist97 — COMMENTED** [review]
Looks good, couple really small things

**Review by jpoist97 — APPROVED** [review]

File: package.json (hunk 4 above)
Comment (jpoist97) [reply package.json:31]: I checked on npm and this is the current version so this is fine

**Review by finlaylp — APPROVED** [review]
Looks great, mostly trusting your coding skill here cuz I didn't understand a couple parts of it.

**Pull request merged**